sdk/python/tests/integration/feature_repos/universal/data_source_creator.py

```diff
@@ -51,6 +52,10 @@ def create_offline_store_config(self) -> FeastConfigBaseModel:
     def create_saved_dataset_destination(self) -> SavedDatasetStorage:
         ...
 
+    @abstractmethod
+    def create_logged_features_destination(self) -> LoggingDestination:
+        ...
```

Comment:
I would create a dummy no-op implementation rather than create an `@abstractmethod` otherwise plugin tests will break when they get this change.